wkcuber/pyproject.toml

```diff
@@ -1,49 +1,27 @@
 [tool.poetry]
 name = "wkcuber"
-version = "0.0.0"  # filled by dunamai
+version = "0.0.0"
 description = "Python package to create, cube, and work with WEBKNOSSOS WKW datasets"
 authors = ["scalable minds <hello@scalableminds.com>"]
 readme = "README.md"
 license = "AGPL-3.0"
 repository = "https://github.com/scalableminds/webknossos-libs"
 homepage = "https://docs.webknossos.org/wkcuber"
-include = ["wkcuber/version.py"]  # included explicitly since it's in .gitignore
+
+[tool.poetry.scripts]
+wkcuber = "wkcuber.main:app"
 
 [tool.poetry.dependencies]
-python = ">=3.8,<3.11"
-attrs = "^22.1.0"
-cattrs = "^22.2.0"
-cluster_tools = { path = "../cluster_tools/", develop = true }
-czifile = "^2019.7.2"
-GitPython = "^3.0.5"
-imagecodecs = ">=2020.5.30"
-natsort = "^6.2.0"
-nibabel = "^5.0.1"
-numpy = "^1.21.0"
-pillow = ">=6.2.1,<9.0.0"
-requests = "^2.22.0"
-scikit-image = "^0.19.3"
-scikit-learn = "^1.0.1"
-scipy = "^1.9.0"
-tifffile = ">=2020.11.26"
+python = ">=3.10,<3.11"
```

Comment:
Is there a reason for dropping Py 3.8 and Py 3.9? The wk-libs support both. I think it would make sense to have the same versions for both packages otherwise you need two virtual envs with different Python versions which lead to confusion.